### ISSUE TITLE
Code Gen Panics When there is `monotonic` is defined but no task defined 

### ISSUE THREAD

**Issue by Gentoli** [rtic-rs/cortex-m-rtic#433]
If an app is defined with `monotonic`
```
#[rtic::app(... , monotonic = rtic::cyccnt::CYCCNT,...)]
```
and no task is defined, the procedural macro fails with 
``` log
help: message: called `Option::unwrap()` on a `None` value
```
on this line:
https://github.com/rtic-rs/cortex-m-rtic/blob/e65a86f6962218b66a14b7ccc40eec7976880eba/macros/src/codegen/pre_init.rs#L79

This is a niche issue but could be encountered when creating new projects. There should be a proper error message for this.

**Comment by korken89** [comment]
Hi, this will disappear with the require of monotonic and will be fixed in `0.6.0-alpha.1`.
Thanks for bringing it to our attention!

**Comment by bradleyharden** [comment]
Just ran into this myself. Looking forward to the next release
